platforms/core-execution/workers/src/test/groovy/org/gradle/workers/internal/IsolatableSerializerRegistryTest.groovy

```diff
@@ -367,6 +367,6 @@ class IsolatableSerializerRegistryTest extends Specification {
     }
 
     enum EnumType {
-        FOO, BAR
+        FOO {}, BAR
```

Comment:
❓ Is that enough to force a subtype? Perhaps adding a `toString` override might make the intention clearer